build.sbt

```diff
@@ -35,7 +33,6 @@ val defaultVersions = Map("firrtl" -> "1.2-SNAPSHOT")
 lazy val commonSettings = Seq (
   organization := "edu.berkeley.cs",
   version := "3.2-SNAPSHOT",
-  git.remoteRepo := "git@github.com:freechipsproject/chisel3.git",
```

Comment:
Are you sure this is part of the `GhpagesPlugin`? I think we want to keep this

Reply:
It does seem this depends on `GhpagesPlugin`. I'll look into reviving the old way to set the SCM repository.